ruby_on_rails/rails_basics/controllers.md

```diff
@@ -103,13 +103,13 @@ In the example above, we saw `... code here to set up a new @post based on form
 
 With the `params` hash!  It acts just like a normal Ruby hash and contains the parameters of the request, stored as `:key => value` pairs.  So how do we get the ID of the post we're asking for? `params[:id]`.  You can access any parameters this way which have "scalar values", e.g. strings, numbers, booleans, `nil`... anything that's "flat".  
 
-Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "test post" and `params[:post_body]` might be "body of post" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
+Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "My Test Post" and `params[:post_body]` might be "Body of post!" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
```

Comment:
To go a step further with making these clearer, we could add "title" to the example

```suggestion
Some forms will submit every field as a top level scalar entry in the params hash, e.g. `params[:post_title]` might be "My Test Post Title" and `params[:post_body]` might be "Body of post!" etc and these you can access with no issues.  You have control over this, as you'll learn in the lessons on forms.
```